test/scripts/generate-react-native-app.sh

```diff
@@ -10,7 +10,7 @@ if [ -d "../backend" ]; then
 else
   echo "Using JHipster for Generation. Both backend and frontend will be generated."
   mkdir ../backend && cd ../backend
-  jhipster --blueprints react-native jdl ../${JHI_REACT_NATIVE_APP_NAME}.jdl \
+  rnhipster --blueprints react-native jdl ../${JHI_REACT_NATIVE_APP_NAME}.jdl \
```

Comment:
Is `--blueprints react-native` still necessary when using `rnhipster`?

Reply:
indeed.